### DISCUSSION TITLE
Windows Package Manager v1.1.2.0

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-create#296] [General]
This release includes [v1.2 Schema](https://github.com/microsoft/winget-cli/tree/master/schemas/JSON/manifests/v1.2.0) support as well as logic to help generate and update manifests for portable packages. We have also made improvements to address issues when submitting manifests to forked repos that are out of sync with the upstream repository.

## Changes:
* 3ea6614241d3a0f5ae5644068230869924c0fbe2 Fix null/empty filename string when downloading package file (#293)
* 99cc214c476b2be3fc9a53776eea781accc0df9c Add support for creating and updating portable manifests (#290)
* 9963de9b4ce33edf81955250d1f7fcbb39ca2e41 Sync fork prior to creating reference and handle NotFoundException (#289)
* b8b1ab5f0327c3d7a0417df5cab380161e3d7b76 always prompt for packageId (#288)
* 49a1fb42f5b7f802d920207b6b48176670b7f6d4 Add custom YamlDotNet emitter that makes all fields with newlines use literal style. (#281)
* 9915d3724472232d12072bbea645f2ac97f79b44 Add support for 1.2 Schema (#270)

<details><summary><b>See More</b></summary>
* d06346339218576f74c809984d9ff7cd41f79e90 Add SBOM task to build pipelines (#267)
* 8928d8c0c59ee75f20a8041c34dc02f47381badf update localization files (#266)
* 8827b8d29ff16e869518ab9e76660d66bd778372 Use AppExecutionAlias in release pipeline (#265)
* 375420fe69ff4b45b80c4526f63ef18cf5c58992 Add `.github/fabricbot.json` (#274)
* 141ec233927b6f1d15937733ddd22fbdc944cd21 increment version to 1.1 (#276)
* bda05d61fbfd9847488d7115fd17208038888306 update localized strings (#275)
* * ea9e220b873eda0ba15484f59a58d3e8b95c23df add newly localized strings (#295)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-create/releases/tag/v1.1.2.0'>Windows Package Manager v1.1.2.0</a>.</em>

**Comment by RuinTD** [comment]
How do portable packages work? I tried specifying a ZIP file as an installer URL, but it just says "Failed to parse the package"
